content/en/md2html-setup.md

```diff
@@ -43,13 +42,30 @@ files:
 - Appendix_L.md
 - Appendix_M.md
 ---
+# Detailed instructions see at https://markdown.standardshub.io/md2html-setup.
```

Comment:
@gocadimic, I understand your question now. The guideline comment needs to be inside the example. People can copy the example and past them on their new index.yaml file.

Reply:
@jpradocueva,
Ok, I will remove it there.